sct_ssh.py

```diff
@@ -433,3 +443,99 @@ def gcp_allow_public(user, test_id):
                      project=info['project_id'],
                      zone=i.zone.split('/')[-1])
         click.echo(click.style(f"set netwrok tag 'sct-allow-public' to {get_name(i)}", fg='green'))
+
+
+@click.command("update-scylla-packages", help="Update ScyllaDB cluster packages ...")
+@click.option("--test-id", type=str, help='Find cluster by test-id', required=True)
+@click.option(
+    "-p", "--packages-location", type=str,
+    help='Location where new packages are placed. Can be local directory, s3:// or gs:// urls')
+@click.option('--backend', type=str, help='Backend to search nodes in. Defaults to aws', default="aws")
+@click.option(
+    '-r', '--region', type=str,
+    help="Cloud region to search nodes in. Defaults to eu-west-1", default="eu-west-1")
+def update_scylla_packages(test_id: str, backend: str, region: str, packages_location: str) -> None:
+    """Update scylla DB packages on selected nodes."""
+    # suppress paramiko 'Connected. Authentication successful!' kind of messages that are logged every second
+    paramiko_logger = logging.getLogger("paramiko")
+    paramiko_logger.setLevel(logging.WARNING)
+
+    if not (packages_location := packages_location or os.getenv('SCT_UPDATE_DB_PACKAGES')):
+        LOGGER.error("No location is provided where new packages are placed")
+        sys.exit(1)
+    if packages_location.startswith('s3') or packages_location.startswith('gs'):
+        packages_location = download_dir_from_cloud(packages_location)
+    if not packages_location.endswith('/'):
+        packages_location += '/'
+
+    instances = select_instance_group(region=region, backends=[backend], test_id=test_id)
+    for instance in instances:
+        tags = get_tags(instance)
+        instance.update({
+            'NodeType': tags['NodeType'],
+            'Name': tags['Name'],
+            'User': tags['RunByUser']
+        })
+
+    if any([i for i in instances if 'db' not in i['NodeType']]):
+        LOGGER.error("Only DB nodes should be selected for updating scylla packages")
+        sys.exit(1)
+
+    nodes = []
+    for instance in instances:
```

Comment:
probably want to run this before opening connection:

```
RemoteCmdRunnerBase.set_default_ssh_transport('libssh2')
```

Reply:
Added defaulting to this transport. This does help to avoid from being spammed by debug messages